perdido/section-styles.js

```diff
@@ -0,0 +1,192 @@
+var blockMar = 30;
+
+window.setupStyles = {
+  // Reset code
+  '*, *:after, *:before': {
+    boxSizing: 'border-box',
+    padding: '0',
```

Comment:
It could but defaultUnit didn't get the update for unitless zeros out to npm yet so I wrapped it in quotes.